cumulusci/cumulusci.yml

```diff
@@ -854,6 +877,7 @@ flows:
                 task: uninstall_managed
 
     install_beta:
+        group: Install / Uninstall
```

Comment:
install_beta belongs in the Org Setup group because it does the full setup including dependencies, installation, and post-install config. It should have been called beta_org